xcube/cli/gen.py

```diff
@@ -90,12 +90,17 @@ def generate_cube(input_files: str,
     Level-2C data cubes may be created in one go or successively in append mode, input by input.
     The input may be one or more input files or a pattern that may contain wildcards '?', '*', and '**'.
     """
+    try:
+        arg_obj = locals()
+    except SystemExit as e:
```

Comment:
A call to ` locals()` does not raise any exception, afaik.

Reply:
Please try understanding the old code first, before changing its structure. You may unintentionally change the logic. Passing `locals()` as arg_obj doesn't make sense, because locals does not have the expected attributes, please look into the implementation of `get_config_dict()`.

B.t.w. where is the new logic that accpets multiple `-c <config>` options?